GliaWidgets/ViewController/Chat/ChatViewController.Mock.swift

```diff
@@ -55,141 +54,8 @@ extension ChatViewController {
             return fileDownload
         }
 
-        let messageUuid = UUID.incrementing
-        let messageId = { messageUuid().uuidString }
-        let fileUuid = UUID.incrementing
-        let fileId = { fileUuid().uuidString }
-        let queueId = UUID.mock.uuidString
-        let operatorAttachmentURL = URL.mock.appendingPathComponent("image").appendingPathExtension("png")
         chatViewModelEnv.fetchChatHistory = { $0(.success([])) }
-        chatViewModelEnv.chatStorage.messages = { _ in
```

Comment:
Do snapshot tests work after deleting mocked messages?

Reply:
I think no, but we will need to update snapshot significantly because lot of changes in UI.

Reply:
Will be good to create ticket for fixing Snapshot test